gradle/libs.versions.toml

```
@@ -33,14 +33,13 @@ kotlinx-serialization-json = { group = "org.jetbrains.kotlinx", name = "kotlinx-
 kotlinx-serialization-protobuf = { group = "org.jetbrains.kotlinx", name = "kotlinx-serialization-protobuf", version.ref = "kotlin-serialization" }
 kotlinx-benchmark = { group = "org.jetbrains.kotlinx", name = "kotlinx-benchmark-runtime", version.ref = "kotlinx-benchmark" }
 
-# plgins
+# plugins
 atomicfu-plugin = { module = "org.jetbrains.kotlinx:atomicfu-gradle-plugin", version.ref = "kotlinx-atomicfu" }
 kotlin-gradle-plugin = { group = "org.jetbrains.kotlin", name = "kotlin-gradle-plugin", version.ref = "kotlin" }
-# Do not update until https://github.com/Kotlin/kotlinx-atomicfu/issues/281 is fixed
-binary-compatibility-validator-plugin = { module = "org.jetbrains.kotlinx:binary-compatibility-validator", version = "0.12.1" }
-dokka-plugin = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version = "1.8.10" }
-
-
+binary-compatibility-validator-plugin = { module = "org.jetbrains.kotlinx:binary-compatibility-validator", version = "0.13.2" }
+dokka-plugin = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version = "1.9.10" }
+maven-publish-plugin = { module = "com.vanniktech:gradle-maven-publish-plugin", version = "0.27.0" }
+ kord-gradle-plugin = { module = "dev.kord:gradle-tools", version = "1.2.0" }
```

```suggestion
kord-gradle-plugin = { module = "dev.kord:gradle-tools", version = "1.2.0" }
```
probably unintended indentation